src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/importcontrol/InputImportControlStrategyOnMismatchOne.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0"?>
 <!DOCTYPE import-control PUBLIC
-    "-//Checkstyle//DTD ImportControl Configuration 1.3//EN"
-    "https://checkstyle.org/dtds/import_control_1_3.dtd">
```

Comment:
this seems outdated and should actually be: `import_control_1_4`. 

the tests should be up to date, right? so this is maybe a gap.